packages/app/src/utils/is-absolute-url.ts

```diff
@@ -0,0 +1 @@
+export const isAbsoluteUrl = (url: string) => /^https?:\/\//.test(url);
```

Comment:
Javascript has this beatiful function keyword:
```suggestion
export function isAbsoluteUrl(url: string) {
  return /^https?:\/\//.test(url);
}
```